test/e2e/common/setup-kubestellar.sh

```diff
@@ -128,7 +128,20 @@ popd
 
 : Waiting for OCM hub to be ready...
 kubectl wait controlplane.tenancy.kflex.kubestellar.org/its1 --for 'jsonpath={.status.postCreateHooks.its-with-clusteradm}=true' --timeout 400s
-kubectl wait -n its1-system job.batch/its-with-clusteradm --for condition=Complete --timeout 400s
+
+# Robust wait for job.batch/its-with-clusteradm
+if ! kubectl wait -n its1-system job.batch/its-with-clusteradm --for condition=Complete --timeout 400s; then
+    echo "ERROR: Timed out waiting for job.batch/its-with-clusteradm to complete. Dumping job and pod status for debugging..." >&2
+    kubectl get job -n its1-system its-with-clusteradm -o yaml || true
+    echo "--- Pods for job its-with-clusteradm ---"
+    kubectl get pods -n its1-system -l job-name=its-with-clusteradm -o wide || true
+    for pod in $(kubectl get pods -n its1-system -l job-name=its-with-clusteradm -o jsonpath='{.items[*].metadata.name}'); do
+        echo "--- Logs for pod $pod ---"
+        kubectl logs -n its1-system "$pod" || true
```

Comment:
The existing E2E workflow already dumps the logs of failed Pods.
The logs of successful Pods are unlikely to be interesting.